joomla-extensions/jedchecker
# Update gpl.ini

### Summary of Changes
Add GPL-compatible and OSI-approved code licenses from:
https://www.gnu.org/licenses/license-list.en.html
https://opensource.org/licenses
https://spdx.org/licenses/

### Testing Instructions
Install, upload any extension (e.g. one with different 3rdparty PHP libraries) and click "Check" button

### Actual result BEFORE applying this Pull Request
Some false-positives could be found.

### Expected result AFTER applying this Pull Request
All licenses are taken into account

### Documentation Changes Required
None (as there is no official documentation page with detailed info on every rules)
